tests/test_plots/test_plots_video.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from unittest.mock import patch, MagicMock
+import numpy as np
+
+from mio.models.frames import NamedVideo
+from mio.plots.video import VideoPlotter
+
+class TestVideoPlotter(unittest.TestCase):
+
+    def setUp(self):
+        # Create a list of NamedFrame instances with dummy data
+        self.named_frames = [
+            NamedVideo(name="Video1", video=[np.random.rand(64, 64) * 255 for _ in range(50)]),
+            NamedVideo(name="Video2", video=[np.random.rand(64, 64) * 255 for _ in range(50)]),
+        ]
+
+    @patch('mio.plots.video.plt.show')
+    @patch('mio.plots.video.plt.subplots')
+    @patch('mio.plots.video.Slider')
+    @patch('mio.plots.video.Button')
+    def test_show_video_with_controls(self, MockButton, MockSlider, mock_subplots, mock_show):
```

Comment:
this doesn't *really* test anything important about this class, but since it's not mission critical and something we will take out and put in a separate GUI package anyway, fine for now.

Reply:
Makes sense. Not exactly sure how to test widgets at the moment, but can do it later if we're isolating this.